libs/explored-types/src/types.ts

```diff
@@ -1,4 +1,4 @@
-import {
+import type {
```

Comment:
Looks good, that lint rule change I mentioned will enforce these explicit type imports.